index.html

```diff
@@ -165,22 +165,52 @@ <h1>My <span>My Portfolio</span></h1>
       <hr class="line" />
       <div class="portfolio-grid">
         <div class="port-grid-items">
-          <img src="" alt="" class="port-image" />
+          <a href="#"
+            ><img
+              src="./assets/images/profile-img.jpg"
+              alt=""
+              class="port-image"
+          /></a>
         </div>
         <div class="port-grid-items">
-          <img src="" alt="" class="port-image" />
+          <a href="#"
+            ><img
+              src="./assets/images/profile-img.jpg"
+              alt=""
+              class="port-image"
+          /></a>
         </div>
         <div class="port-grid-items">
-          <img src="" alt="" class="port-image" />
+          <a href="#"
+            ><img
+              src="./assets/images/profile-img.jpg"
+              alt=""
+              class="port-image"
+          /></a>
         </div>
         <div class="port-grid-items">
-          <img src="" alt="" class="port-image" />
+          <a href="#"
+            ><img
+              src="./assets/images/profile-img.jpg"
+              alt=""
+              class="port-image"
+          /></a>
         </div>
         <div class="port-grid-items">
-          <img src="" alt="" class="port-image" />
+          <a href="#"
+            ><img
+              src="./assets/images/profile-img.jpg"
+              alt=""
+              class="port-image"
+          /></a>
         </div>
         <div class="port-grid-items">
-          <img src="" alt="" class="port-image" />
+          <a href="#"
+            ><img
+              src="./assets/images/profile-img.jpg"
+              alt=""
+              class="port-image"
+          /></a>
         </div>
       </div>
     </section>
```

Comment:
You have done really great especially using the HTML semantic tags. However, I would suggest you limit your use of non-semantic tags. So far, so good. You are doing well.